app.py

```
@@ -161,7 +200,7 @@ def generate_newapi_param(
 
 
 def request_newsdataapi(params: NewsDataApiParam) -> Union[SearchSuccess, SearchError]:
-    api_key = os.getenv("NEWSDATAAPI_KEY")
+    api_key = os.environ.get("NEWSDATAAPI_KEY")

```

I would prefer `os.environ["NEWSDATAAPI_KEY"]` as it will raise `KeyError` if it does not exist instead of silently return `None`.